sendrecv/gst/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM maxmcd/gstreamer:1.14-buster
+
+RUN apt-get install -y libjson-glib-dev
+# RUN apk update
+# RUN apk add json-glib-dev libsoup-dev
+
+WORKDIR /opt/
+COPY . /opt/
+
+RUN sed -i 's/wss:\/\/webrtc.nirbheek.in:8443/ws:\/\/signalling:8443/g' \
+    /opt/webrtc-sendrecv.c
+RUN sed -i 's/strict_ssl = TRUE/strict_ssl = FALSE/g' \
+    /opt/webrtc-sendrecv.c
```

Comment:
Here, too. I think it would be generally useful for people (while running local servers) if we do that.